docs/development/guide.md

```diff
@@ -59,6 +59,12 @@ Launch the development server with the hot reloading functionality that allows a
 npm start
 ```
 
+## Launch a development environment with Gitpod
```

Comment:
You wrote that section not in appropriate place, you should write it after sentence:

> You can access the live development server at [localhost:3000](http://localhost:3000/).

Also you should change gitpod link to open `next` branch:

```
https://gitpod.io/#https://github.com/asyncapi/asyncapi-react/tree/next
```